webpackjs/components/Calcs/CalcComponent.js

```diff
@@ -0,0 +1,155 @@
+import React from "react";
+
+const CalcComponent = ({ objectProps, inputWidth, inputLinearFeet, inputThickness, inputRPMS, output, getOutputWidth, getOutputLinearFeet, getOutputThickness, getOutputRPMS }) => {
```

Comment:
Are we using Prettier yet? If not, can we make an issue to add it to this repo @tcwcodes. @mddilley can help guide based how he's has already added prettier configs to other repos.